containers/Balancer.ts

```diff
@@ -33,10 +33,23 @@ interface PoolTokenQuery {
   balance: string;
 }
 
-const YIELD_TOKENS = [
-  "0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5", // Sep20
-  "0xB2FdD60AD80ca7bA89B9BAb3b5336c2601C020b4", // Oct20
-];
+interface YIELD_TOKEN {
+  [key: string]: string[];
+}
+const YIELD_TOKENS: YIELD_TOKEN = {
+  "0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5": [
+    "0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5".toLowerCase(),
+    "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48".toLowerCase(),
+  ], // yUSDETH-SEP20
+  "0xB2FdD60AD80ca7bA89B9BAb3b5336c2601C020b4": [
+    "0xb2fdd60ad80ca7ba89b9bab3b5336c2601c020b4".toLowerCase(),
```

Comment:
What are the two elements in these arrays? Would it be clearer to make them key value pairs?

Reply:
Are they [0] = synth, [1] = collateral?

Reply:
In this, the key is the synthetic token and the value is the associated balancer pool pair in the correct order for querying. It cant be a key-value pair as sometimes the ordering in the query within balancer swaps direction. For some of our tokens, this is `[syntheticAddress,USDCAddress]` buy for others, like ren this is `[USDCAddress,syntheticAddress]`.

does that make sense?

Reply:
> Are they [0] = synth, [1] = collateral?

not always; this is the reason I had to add this mapping object like this. For the ren synthetic a query that works looks like this:
```
{
    pools(where: {tokensList: ["0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48","0x208d174775dc39fe18b1b374972f77ddec6c0f73"]}) {
      swapFee
      liquidity
      totalSwapVolume
      tokens {
        address
        balance
      }
      joinsCount
      exitsCount
      swapsCount
      totalShares
      shares(first:1000, where: { balance_gt: 0}) {
        userAddress {
          id
        }
        balance
      }
    }
  }
```
This has `[USDCAddress,syntheticAddress]` in the `where` clause. For the other yield tokens this would be swapped.

Reply:
This makes sense. Would it be accurate (and more readable) to make the keys `token0` and `token1`?

Reply:
Ye, sure. done. PTAL